backend/src/main/java/com/onwelo/practice/bts/entity/BankAccount.java

```diff
@@ -1,15 +1,18 @@
 package com.onwelo.practice.bts.entity;
 
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
 import javax.persistence.*;
-import java.util.Collection;
+import java.util.List;
 
 @Entity
-@Table(name = "bank_account")
```

Comment:
Wolałbym żeby od razu ponazywać tabele i kolumny. Potem przy ewentualnej zmianie bazy na mySQL albo coś podobnego będzie mniej roboty. Poza tym to jest dobry zwyczaj.